### ISSUE TITLE
Sharing: deprecate wp-admin > Settings > Sharing page

### ISSUE THREAD

**Issue by jeherve** [Automattic/jetpack#21566]
### Impacted plugin

Jetpack

### What

Let's consolidate Sharing / Like settings by deprecating the Settings > Sharing page in wp-admin. There are other places where folks can manage their settings.

### How

It's worth noting that even though the page does not include any Publicize or Verification tools settings, it is still needed for those modules to work, so this would have to be taken into account when deprecating the page:
https://github.com/Automattic/jetpack/blob/12c1ec934857a0c0fa2afe2ddecbddb7cdbf2e4e/projects/plugins/jetpack/modules/publicize/ui.php#L37-L40
https://github.com/Automattic/jetpack/blob/c52e77b1b4151e53a430b2d6ceb0adfb4e468faf/projects/plugins/jetpack/_inc/lib/class.jetpack-keyring-service-helper.php#L56

We will also need to account for options that are currently only available in that screen:

- The option to add custom sharing services.
- The option to use your own custom assets to manage the look of the sharing buttons.

![image](https://user-images.githubusercontent.com/426388/139219686-44e0562b-38fd-4200-9a3e-685887544d34.png)
![image](https://user-images.githubusercontent.com/426388/139219711-b3be0833-f26a-46eb-b78d-e9624549347a.png)

--------

Previous art on this: #11314

Discussion: 

p8oabR-K9-p2/#comment-5540

**Comment by samiff** [comment]
> We will also need to account for options that are currently only available in that screen

Since the sharing module doesn't require a user connection, would we would need to maintain configuring the sharing buttons from wp-admin and not linking to Calypso for that?

**Comment by jeherve** [comment]
> would we would need to maintain configuring the sharing buttons from wp-admin and not linking to Calypso for that?

Yep, correct. This is where having configuration options in the Jetpack dashboard would be better, if we want to deprecate the old sharing settings.
